package.json

```diff
@@ -34,13 +34,14 @@
         "build:prod": "jlpm clean && jlpm build:lib:prod && jlpm build:labextension",
         "build:labextension": "jupyter labextension build .",
         "copy:landing:template": "cp templates/index.html dist/index.html",
+        "patch:lab:index": "patch -p1 --forward < templates/customise-title-and-spinner.patch || patch -p1 -R --dry-run < templates/customise-title-and-spinner.patch",
```

Comment:
This is a limitation in `patch` itself; https://stackoverflow.com/q/23395363, and I am not keen on using `git am` or `git apply` when `patch` can do the trick.